instrumentation/jetty/jetty-11.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jetty/v11_0/Jetty11HandlerAdvice.java

```diff
@@ -22,30 +23,34 @@ public static void onEnter(
       @Advice.This Object source,
       @Advice.Argument(2) HttpServletRequest request,
       @Advice.Argument(3) HttpServletResponse response,
+      @Advice.Local("otelRequest") ServletRequestContext<HttpServletRequest> requestContext,
       @Advice.Local("otelContext") Context context,
       @Advice.Local("otelScope") Scope scope) {
 
-    Context attachedContext = tracer().getServerContext(request);
+    Context attachedContext = helper().getServerContext(request);
```

Comment:
This should be jetty specific. We instrument all implementations of `org.eclipse.jetty.server.Handler` and need to make sure that we start span only in the outermost one. I believe that we should use `shouldStart` to be consistent with other instrumentations, but I suspect that we can't just replace the existing code with it because `shouldStart` counts suppressed spans and using it for jetty would blow that number up.